### PR TITLE
Extract processBlock method and query getter

### PR DESCRIPTION
**Pull request by julianrubisch**
Houekeeping 🧹

### CODE REVIEW

File: javascript/elements/updates_for_element.js [hunk 1]
```diff
@@ -134,6 +134,10 @@ export default class UpdatesForElement extends SubscribingElement {
     )
   }
 
+  get query () {
+    return `updates-for[identifier="${this.getAttribute('identifier')}"]`
```
Comment (marcoroth) [inline javascript/elements/updates_for_element.js:138]: Can't you use `this.identifier` here too?
```suggestion
    return `updates-for[identifier="${this.identifier}"]`
```

File: javascript/elements/updates_for_element.js [hunk 2]
```diff
@@ -41,9 +41,8 @@ export default class UpdatesForElement extends SubscribingElement {
   }
 
   async update (data) {
```
Comment (marcoroth) [inline javascript/elements/updates_for_element.js:43]: I think you can remove the `async` from the `update()` function now that you moved the async part to the `processBlock()` function

Comment (julianrubisch) [reply javascript/elements/updates_for_element.js:43]: good spot!

File: javascript/elements/updates_for_element.js (hunk 1 above)
Comment (julianrubisch) [reply javascript/elements/updates_for_element.js]: 🙏

**Pull request merged**